src/main/java/frc/robot/RobotContainer.java

```diff
@@ -319,10 +335,12 @@ private void configureInputs() {
 
     operatorOI.getShooterRollerButton()
         .whileActiveContinuous(new RunShooterRoller(shooterRoller).alongWith(new RunHopper(hopper)));
-    operatorOI.getShooterUnstickButton().whileActiveContinuous(new FeedUnstick(shooterRoller, hopper));
+    operatorOI.getShooterUnstickButton()
+        .whileActiveContinuous(new FeedUnstick(shooterRoller, hopper, operatorOI::updateLED));
 
-    operatorOI.getIntakeExtendButton().whenActive(new InstantCommand(intake::extend, intake));
-    operatorOI.getIntakeRetractButton().whenActive(new InstantCommand(intake::retract, intake));
+    operatorOI.getIntakeExtendButton().whenActive(() -> intake.extend());
+    operatorOI.getIntakeRetractButton().whenActive(() -> intake.retract());
```

Comment:
```suggestion
    operatorOI.getIntakeExtendButton().whenActive(intake::extend, intake);
    operatorOI.getIntakeRetractButton().whenActive(intake::retract, intake);
```
These should require the intake (and method references are a little cleaner than lambdas here)